tests/MongoDB.Driver.Legacy.Tests/MongoCollectionTests.cs

```diff
@@ -2073,15 +2073,17 @@ public void TestGroupWithFinalizeFunction()
             _collection.Insert(new BsonDocument("x", 3));
             _collection.Insert(new BsonDocument("x", 3));
             _collection.Insert(new BsonDocument("x", 3));
-
+            
```

Comment:
Super minor nit: there's some extraneous whitespace before and after the `#pragma` statements in this file that could be removed :)